docs/dao/governance/process.mdx

```diff
@@ -15,7 +8,7 @@ This document is a suggested process for developing and advancing ENS Governance
 
 [discuss.ens.domains](https://discuss.ens.domains) is a Discourse forum for governance-related discussion. Community members must register for an account before sharing or liking posts. Registering for the forum allows community members to post in the general forum; for access to the working groups, fill out the [participant request form](https://airtable.com/shrv2xP39SmuCcd5j).
 
-There are four workstream categories: [Meta-Governance](https://discuss.ens.domains/c/meta-governance/28), [Community](https://discuss.ens.domains/c/community/12), [ENS Ecosystem](https://discuss.ens.domains/c/ens-ecosystem/32), and [Public Goods](https://discuss.ens.domains/c/public-goods/37). Each category has subcategories for each of the steps of the governance process described below.
+There are three workstream categories: [Meta-Governance](https://discuss.ens.domains/c/meta-governance/28), [Community](https://discuss.ens.domains/c/community/12), and [ENS Ecosystem](https://discuss.ens.domains/c/ens-ecosystem/32). Each category has subcategories for each of the steps of the governance process described below.
```

Comment:
```suggestion
There are three workstream categories: [Meta-Governance](https://discuss.ens.domains/c/meta-governance/28), [Public Goods](https://discuss.ens.domains/c/public-goods/37), and [ENS Ecosystem](https://discuss.ens.domains/c/ens-ecosystem/32). Each category has subcategories for each of the steps of the governance process described below.
```

Reply:
Made the suggested corrections. 

The Meta object appears on other files in the governance folder, but as long as it’s all working as intended I won’t edit those in a separate PR.